tests/x11regressions/evolution/evolution_smoke.pm

```diff
@@ -32,8 +27,28 @@ sub run {
     else {
         send_key $self->{next};
     }
+
     assert_screen "evolution_wizard-done";
     send_key "alt-a";
+}
+
+sub run {
+    my $self        = shift;
+    my $mail_box    = 'nooops_test3@aim.com';
+    my $mail_passwd = 'opensuse';
+
+    mouse_hide(1);
+    evolution_wizzard($self, $mail_box);
+
+    # Give test time to connect to remote server
+    wait_still_screen 10;
+    # Proceed with soft-failure if it is still connecting
+    if (check_screen("evolution_smoke-detect-folders-scanning")) {
```

Comment:
Both the `wait_still_screen` and `check_screen` should be combinable by a multi-tag `assert_screen` with `match_has_tag` and a properly defined needle, e.g.

```perl
assert_screen([qw(evolution_smoke-detect-folders-scanning evolution_mail-auth)]);
if (match_has_tag 'evolution_smoke-detect-folders-scanning') {
    …
    assert_screen 'evolution_mail-auth';
}
```

Reply:
@okurz Screen with "folders-scanning" is there in both cases,  even when it works fine. It is there just for short time. Issue happens when that screen is there for longer time(not responding server). It means that multi-tag match is not applicable. Does it make sense?

Reply:
I suspect there must me something that can be matched on the screen to detect the difference. Can you share a URL to the specific screenshots we need to care about here?

Reply:
This is the place where it hangs,  https://openqa.suse.de/tests/1250282#step/evolution_smoke/12 . If it is fine, it is there for short time. If there is an issue,  it will hang.

Reply:
@okurz maybe i found, how to do it better. I will try something similar to what is in function "ensure_installed". Switching to wip...